{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright Â© Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An update for python-rtslib is now available for Red Hat Enterprise Linux\n8.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "For detailed information on changes in this release, see the Red Hat\nEnterprise Linux 8.3 Release Notes linked from the References section.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHEA-2020:4505",
				"url": "https://access.redhat.com/errata/RHEA-2020:4505"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.3_release_notes/index",
				"url": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.3_release_notes/index"
			},
			{
				"category": "external",
				"summary": "1820170",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1820170"
			},
			{
				"category": "external",
				"summary": "1849651",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1849651"
			},
			{
				"category": "external",
				"summary": "1858856",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1858856"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2020/rhea-2020_4505.json"
			}
		],
		"title": "Red Hat Enhancement Advisory: python-rtslib bug fix and enhancement update",
		"tracking": {
			"current_release_date": "2024-09-16T04:14:39+00:00",
			"generator": {
				"date": "2024-09-16T04:14:39+00:00",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "3.33.3"
				}
			},
			"id": "RHEA-2020:4505",
			"initial_release_date": "2020-11-04T02:15:29+00:00",
			"revision_history": [
				{
					"date": "2020-11-04T02:15:29+00:00",
					"number": "1",
					"summary": "Initial version"
				},
				{
					"date": "2020-11-04T02:15:29+00:00",
					"number": "2",
					"summary": "Last updated version"
				},
				{
					"date": "2024-09-16T04:14:39+00:00",
					"number": "3",
					"summary": "Last generated version"
				}
			],
			"status": "final",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "Red Hat Enterprise Linux BaseOS (v. 8)",
								"product": {
									"name": "Red Hat Enterprise Linux BaseOS (v. 8)",
									"product_id": "BaseOS-8.3.0.GA",
									"product_identification_helper": {
										"cpe": "cpe:/o:redhat:enterprise_linux:8::baseos"
									}
								}
							}
						],
						"category": "product_family",
						"name": "Red Hat Enterprise Linux"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-rtslib-0:2.1.73-2.el8.src",
								"product": {
									"name": "python-rtslib-0:2.1.73-2.el8.src",
									"product_id": "python-rtslib-0:2.1.73-2.el8.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-rtslib@2.1.73-2.el8?arch=src"
									}
								}
							}
						],
						"category": "architecture",
						"name": "src"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python3-rtslib-0:2.1.73-2.el8.noarch",
								"product": {
									"name": "python3-rtslib-0:2.1.73-2.el8.noarch",
									"product_id": "python3-rtslib-0:2.1.73-2.el8.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-rtslib@2.1.73-2.el8?arch=noarch"
									}
								}
							},
							{
								"category": "product_version",
								"name": "target-restore-0:2.1.73-2.el8.noarch",
								"product": {
									"name": "target-restore-0:2.1.73-2.el8.noarch",
									"product_id": "target-restore-0:2.1.73-2.el8.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/target-restore@2.1.73-2.el8?arch=noarch"
									}
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-rtslib-0:2.1.73-2.el8.src as a component of Red Hat Enterprise Linux BaseOS (v. 8)",
					"product_id": "BaseOS-8.3.0.GA:python-rtslib-0:2.1.73-2.el8.src"
				},
				"product_reference": "python-rtslib-0:2.1.73-2.el8.src",
				"relates_to_product_reference": "BaseOS-8.3.0.GA"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-rtslib-0:2.1.73-2.el8.noarch as a component of Red Hat Enterprise Linux BaseOS (v. 8)",
					"product_id": "BaseOS-8.3.0.GA:python3-rtslib-0:2.1.73-2.el8.noarch"
				},
				"product_reference": "python3-rtslib-0:2.1.73-2.el8.noarch",
				"relates_to_product_reference": "BaseOS-8.3.0.GA"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "target-restore-0:2.1.73-2.el8.noarch as a component of Red Hat Enterprise Linux BaseOS (v. 8)",
					"product_id": "BaseOS-8.3.0.GA:target-restore-0:2.1.73-2.el8.noarch"
				},
				"product_reference": "target-restore-0:2.1.73-2.el8.noarch",
				"relates_to_product_reference": "BaseOS-8.3.0.GA"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2020-14019",
			"cwe": {
				"id": "CWE-282",
				"name": "Improper Ownership Management"
			},
			"discovery_date": "2020-06-19T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "1854723"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A flaw was found in Open-iSCSI rtslib-fb through versions 2.1.72, where it has weak permissions for /etc/target/saveconfig.json because the shutil.copyfile, instead of shutil.copy is used, and permissions are not preserved upon editing. This flaw allows an attacker with prior access to /etc/target/saveconfig.json to access a later version, resulting in a loss of integrity, depending on their permission settings. The highest threat from this vulnerability is to confidentiality.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "python-rtslib: weak permissions for /etc/target/saveconfig.json",
					"title": "Vulnerability summary"
				},
				{
					"category": "other",
					"text": "Red Hat Ceph Storage 2 and 3 are not affected because within the affected method, shutil.copyfile is not used. However, the affected method, save_to_file is outdated and contains a race condition. Hence, this issue has been rated as having a security impact of low.",
					"title": "Statement"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"BaseOS-8.3.0.GA:python-rtslib-0:2.1.73-2.el8.src",
					"BaseOS-8.3.0.GA:python3-rtslib-0:2.1.73-2.el8.noarch",
					"BaseOS-8.3.0.GA:target-restore-0:2.1.73-2.el8.noarch"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2020-14019"
				},
				{
					"category": "external",
					"summary": "RHBZ#1854723",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1854723"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2020-14019",
					"url": "https://www.cve.org/CVERecord?id=CVE-2020-14019"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2020-14019",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14019"
				}
			],
			"release_date": "2020-06-19T00:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"BaseOS-8.3.0.GA:python-rtslib-0:2.1.73-2.el8.src",
						"BaseOS-8.3.0.GA:python3-rtslib-0:2.1.73-2.el8.noarch",
						"BaseOS-8.3.0.GA:target-restore-0:2.1.73-2.el8.noarch"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHEA-2020:4505"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "LOW",
						"attackVector": "LOCAL",
						"availabilityImpact": "LOW",
						"baseScore": 6.6,
						"baseSeverity": "MEDIUM",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "LOW",
						"privilegesRequired": "LOW",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:L",
						"version": "3.1"
					},
					"products": [
						"BaseOS-8.3.0.GA:python-rtslib-0:2.1.73-2.el8.src",
						"BaseOS-8.3.0.GA:python3-rtslib-0:2.1.73-2.el8.noarch",
						"BaseOS-8.3.0.GA:target-restore-0:2.1.73-2.el8.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Moderate"
				}
			],
			"title": "python-rtslib: weak permissions for /etc/target/saveconfig.json"
		}
	]
}
